script/Ciel/Day 7/07_01_CIEL07_1_3.txt

```diff
@@ -495,7 +495,7 @@ In one smooth motion, I uncross my legs and sidle over to Senpai.
 -- 「なぜって、男はみんなそうですけど。女の子の装飾品を剥がすのとか大好きだし。うん、ゆで卵を剥く感じというか、それが末端であるほど興奮しますよね先輩」
 // The fuck am I reading
 "Why? This is just how boys are.
- We simply love to pluck the ornaments off of girls.
+ We simply love to pluck the ornaments off girls.
```

Comment:
I'd personally say `off of` here, which is apparently the American way to do it anyway